scripts/ebeam_scan_to_multiprobe.py

```diff
@@ -7,36 +7,61 @@
 
 Copyright © 2019 Thera Pals, Delmic
 
-This script provides a command line interface for aligning the ebeam scan to multiprobe.
+This script provides a command line interface for aligning the ebeam scan to
+multiprobe.
+
+This script implements the following steps:
+* Acquire image of multiprobe to determine angle of multiprobe.
+* Move multiprobe on single axis with deflectors (by changing DC offset on scan
+deflector AWG) to obtain angle of movement.
+* Calculate angle of movement relative to angle of multiprobe.
+* Match angle of movement to angle of multiprobe by changing scan rotation in
+XT/SEM PC.
+
+Information on how to run this can be found in the sonic-engineering repo:
+sonic-engineering/doc/useful commands.txt
+
+To use this script first run the following line or add it at the end of .bashrc.
+export PYTHONPATH="$HOME/development/sonic-engineering/scanwaveform/:$PYTHONPATH"
```

Comment:
For readability, I'd recommend to even make `scanwaveform` a package (by adding an empty `__init__.py` file), and then you can type `from scanwaveform import dcoffset`.